astropy/io/fits/scripts/fitsdiff.py

```diff
@@ -252,7 +266,8 @@ def main():
         opts.ignore_keywords = []
         opts.ignore_comments = []
         opts.ignore_fields = []
-        opts.tolerance = 0.0
```

Comment:
@MSeifert04 , it appears it is now saved to `rtol`, so this does not exist anymore.